ui/js/graphs.js

```diff
@@ -705,18 +705,22 @@ perf.controller('GraphsCtrl', [
       });
   }]);
 
-perf.controller('TestChooserCtrl', function($scope, $modalInstance, $http,
-                                            projects, optionCollectionMap,
-                                            timeRange, thServiceDomain,
-                                            PhSeries, defaultProjectName,
-                                            defaultPlatform) {
+perf.controller('TestChooserCtrl', function TestChooserCtrl($scope, $modalInstance, $http, 
+               projects, optionCollectionMap, timeRange, thServiceDomain, ThRepositoryModel, 
+               thDefaultRepo, PhSeries, defaultProjectName, defaultPlatform) {
   $scope.timeRange = timeRange;
   $scope.projects = projects;
   if (defaultProjectName) {
     $scope.selectedProject = _.findWhere(projects, {name: defaultProjectName});
   } else {
-    $scope.selectedProject = projects[0];
+    var ret = ThRepositoryModel.getRepo(thDefaultRepo);
```

Comment:
Again, we need to use findWhere here for the same reasons as the compare chooser. If you do what I suggest you can just reuse the findWhere above.